### PR TITLE
Issue 2 user can view their entries

### PR DESCRIPTION
**Pull request by hsu-isaac**

### CODE REVIEW

**Comment by hsu-isaac** [comment]
issue-2

**Comment by hsu-isaac** [comment]
issue-2-mobile

File: js/main.js [hunk 1]
```diff
@@ -22,4 +30,90 @@ function submit(event) {
...
+
+function switchtoForm(event) {
+  data.view = 'entry-form';
+  entryForm.setAttribute('class', 'entry-form');
+  entries.setAttribute('class', 'entries hidden');
+}
+
+if (data.view === 'entry-form') {
+  entryForm.setAttribute('class', 'entry-form');
+  entries.setAttribute('class', 'entries hidden');
```
Comment (codingcodymiller) [inline js/main.js]: You should call `switchToForm` instead

File: js/main.js [hunk 2]
```diff
@@ -22,4 +30,90 @@ function submit(event) {
...
+} else if (data.view === 'entries') {
+  entryForm.setAttribute('class', 'entry-form hidden');
+  entries.setAttribute('class', 'entries');
+  if (data.entries.length > 0) {
+    noEntries.setAttribute('class', 'no-entries hidden');
+  } else {
+    entryForm.setAttribute('class', 'entries hidden');
+  }
```
Comment (codingcodymiller) [inline js/main.js]: You should call `switchToEntries` instead

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**